keanu-python/keanu/algorithm/sampling.py

```diff
@@ -13,25 +16,37 @@
               'NUTS': k.jvm_view().NUTS,
               'hamiltonian': k.jvm_view().Hamiltonian}
 
-def sample(net, sample_from, algo='metropolis', draws=500, drop=0, down_sample_interval=1):
+def sample(net: BayesNet,
+           sample_from: Iterable[Vertex],
+           algo: str = 'metropolis',
+           draws: int = 500,
+           drop: int = 0,
+           down_sample_interval: int = 1) -> Dict[Tuple[Any, ...], List[numpy_types]]:
```

Comment:
I imagine we'll use this typing for `samples`: `Dict[Tuple[Any, ...], List[numpy_types]` quite a lot in the codebase. Can we create a predefined type for it so we don't have to write it out each time? It's used three times in this file alone.

Can we define something so that we can do:


```
down_sample_interval: int = 1) -> Samples:
```

where Samples is `Dict[Tuple[Any, ...], List[numpy_types]]:`